.editorconfig

```diff
@@ -3,9 +3,7 @@
 # top-most EditorConfig file
 root = true
 
-# Unix-style newlines with a newline ending every file
 [*]
-end_of_line = lf
```

Comment:
I see the commit note on this change is to fix the mixed line endings errors on windows -- but I assume that's just an editor issue with an editor ignoring this setting or a git issue with git reporting the issue.

However, the change itself is innocuous since we're committing to git and ultimately we're going to standardize line endings regardless of editorconfig once something is committed. I just wanted to note that for clarity, the stated commit message isn't really an issue with editorconfig.